src/promptflow/promptflow/_sdk/_start_trace.py

```diff
@@ -0,0 +1,38 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import os
+import uuid
+
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+from promptflow._sdk._constants import TRACE_SESSION_ID_ENV_VAR
+
+
+def start_trace():
```

Comment:
need to add docstring and note this is preview api

Reply:
I will add the docstring; meanwhile, I think maybe we don't need to expose it as `pf.start_trace()` in this PR? we can expose it after we are all ready.

Reply:
added, and hide for now